x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_detail.tsx

```diff
@@ -222,8 +222,11 @@ export const RelatedEventDetail = memo(function RelatedEventDetail({
       {
         text: (
           <>
-            <EuiI18nNumber value={totalCount} />
-            {/* Non-breaking space->*/ ` ${eventsString}`}
+            <FormattedMessage
+              id="xpack.securitySolution.enpoint.resolver.panel.relatedEventDetail.numberOfEvents"
+              values={{ totalCount }}

```

Comment:
btw, I'm pretty sure you can pass in JSX as values here as well, so you could do:
```
values={{ totalCount: <EUII18nNumber value={totalCount} /> }}
```